superset-embedded-sdk/package.json

```
@@ -22,7 +22,7 @@
   "module": "lib/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsc & babel src --out-dir lib --extensions '.ts,.tsx' & webpack --mode production",
+    "build": "tsc ; babel src --out-dir lib --extensions '.ts,.tsx' ; webpack --mode production",
```

I didn't realize this before but `&` actually starts a process in the background instead of working like `Promise.all`

---

```suggestion
    "build": "tsc && babel src --out-dir lib --extensions '.ts,.tsx' && webpack --mode production",
```

I suggest use ``&&`` to execute that in series only if the predecessor was successful. In other words, stop as soon as an error occurs.

---

These aren't really dependent on each other's result. Running each of them regardless of the others' status can output more useful information in the case of an error. Ideally I'd like to run them in parallel and wait for all to complete before the command, but I am not familiar with a convenient way to do that without writing a script, and that seems like overkill since these all run in about a second.